tsconfig.json

```diff
@@ -4,7 +4,7 @@
         "module": "node16",
         "moduleResolution": "node16",
         "declaration": false,
-        "target": "es2020",
+        "target": "es2021",
```

Comment:
I'd prefer not to change the target in a minor/patch version if I can help it. Why do you need the 2021 target?

Reply:
Sorry, I was using `.replaceAll()` and TypeScript was giving me problems if I didn't use a version later than `2020`. But it's fixed now.